devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -0,0 +1,138 @@
+/**
+ *
+ *	Copyright 2019 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ */
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+    definition(name: "ZigBee Window Shade", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.blind") {
+        capability "Actuator"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Window Shade"
+        capability "Health Check"
+        capability "Switch Level"
+
+        command "pause"
+
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0102", outClusters: "0019", model: "E2B0-KR000Z0-HA", deviceJoinName: "SOMFY Blind Controller/eZEX", vid:"SmartThings-smartthings-Somfy_WindowShade" // SY-IoT201-BD
```

Comment:
nit: add spacer after `vid`. Also, could we just get metadata for this DTH instead of device specific metadata?

Reply:
agree with tom that this vid should just be moved into the device definition

Reply:
I thought that Somfy named vid is not good in the definition because we think this DTH as a generic zigbee window shade DTH.
Should I remove the vid from the fingerprint and move it to the definition with `generic-shade` or `SmartThings-smartthings-Somfy_WindowShade` ?

Reply:
What they mean is instead of 

```
    definition(name: "ZigBee Window Shade", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.blind") {

...

        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0102", outClusters: "0019", model: "E2B0-KR000Z0-HA", deviceJoinName: "SOMFY Blind Controller/eZEX", vid:"SmartThings-smartthings-Somfy_WindowShade" // SY-IoT201-BD
```
to use
```
    definition(name: "ZigBee Window Shade", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.blind", mnmn: "SmartThings", vid: "generic-shade") {

...

        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0102", outClusters: "0019", model: "E2B0-KR000Z0-HA", deviceJoinName: "SOMFY Blind Controller/eZEX" // SY-IoT201-BD
```

Reply:
Got it. Thank you :)